server/google_apis/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Use Google's official Dart image.
```

Comment:
Isn't this identical to the `dart-runtime` image?

Reply:
No! See https://github.com/dart-lang/dart_docker/issues/69

It really matters when doing the default deploy w/ cloud run!